packages/plugin-conversation/src/share-activity.js

```diff
@@ -75,23 +75,25 @@ const ShareActivity = SparkPlugin.extend({
    * Adds an additional file to the share and begins submitting it to spark
    * files
    * @param {File} file
+   * @param {Object} options
+   * @param {Object} options.actions
    * @returns {EventEmittingPromise}
    */
-  add(file) {
+  add(file, options) {
     let upload = this.uploads.get(file);
     if (upload) {
       return upload[PROMISE_SYMBOL];
     }
 
     const emitter = new EventEmitter();
 
-    upload = {
+    upload = Object.assign({
```

Comment:
yep. Object.assign is the way to go. Most (though not all) lodash methods that operate on collections now exist in the core language in some way.

map, reduce, filter, includes, find, findIndex are all part of array now.